README.md

```diff
@@ -48,3 +55,48 @@ bundle install
 bundle exec rake test
 bundle exec jekyll serve
 ```
+
+## Archiving the site
+
+We archive snapshots of the whole site for every release, which allows our customers to time travel and review
+older docs. This is very valuable if they're running older versions and need to look something up.
+Archives are maintained at https://archive.istio.io.
+
+To create an archive version, you need to follow these steps:
+
+- Go to https://google.com/cse and create a new search engine for the archive site. After you've gone through the hoops necessary
+to get access to this site, you'll see there are many search engines already created. We create one search engine
+per site archive, which makes it possible for customers to search within this specific version of the site. You will need
+to create a new search engine. Just create a new engine and specify that you'd like to search the url https://archive.istio.io/vXX
```

Comment:
I don't think we should make 1 engine per version, seems wasteful and unnecessary (not counting an extra step/barrier every month)